pkg/syfs/syfs.go

```diff
@@ -75,6 +77,12 @@ func configDir(dir string) string {
 		} else {
 			homedir = user.HomeDir
 		}
+	} else if homedir != nonexistentDir && !fs.IsReadable(homedir) {
```

Comment:
This shouldn't be an `else`, because the code just above might set the home directory to a non-readable place.

Reply:
yeah, correctly. Thanks for the correction.